pyproject.toml

```diff
@@ -69,7 +69,7 @@ filterwarnings = ["ignore::DeprecationWarning"]
 [tool.pylint.master]
 # Since the data store is part of an R&D project, it makes sense to move fast
 # and leave TODOs for later.
-disable=["fixme", "too-few-public-methods", "broad-exception-raised"]
+disable=["fixme", "too-few-public-methods", "broad-exception-raised", "import-outside-toplevel", "R"]
```

Comment:
@fivegrant Please scope this change. This disables all ["Refactor" messages](https://pylint.readthedocs.io/en/latest/user_guide/messages/messages_overview.html#refactor)